lib/grid.rb

```diff
@@ -9,10 +11,22 @@ def initialize(cells = {})
     @cells = cells
   end
 
+  def game_over?
```

Comment:
This made sense here when there was just it's fullness to consider, but now has a lot extra knowledge.